### DISCUSSION TITLE
Delete all versions - No Recycle

### DISCUSSION THREAD

**Discussion by TheRealStriff** [Barbarur/NovaPoint#30] [Ideas]
Don't use this option much, but when I did, realised unlike the Keep few versions,there's no option to "Recycle" or "Delete"
Seems to Recycle, be great if this too had an option to just Delete.

Cheers

**Comment by Barbarur** [comment]
That's is because currently there is no API for "Recycle all" versions.

When keeping few versions, the delete/recycle of versions is done one by one per version. Meanwhile Delete all is a single action and the server manage the deletion from its side.

Current alternative would be to use the option to keep few versions, set it to 0 versions and select recycle.

I'll see if there is a better way to display the different options from the UI to bring more clarity.

**Reply by TheRealStriff** [reply]
Thanks for the explanation.
Makes sense,,,, Cheers
